@helen There are two ESLint errors:
```shell
autocompleters/index.js
  121:22  error  'textNode' is defined but never used                     no-unused-vars
  126:16  error  Unknown property 'class' found, use 'className' instead  react/no-unknown-property
```
It's severely obfuscated both locally and in the CI jobs amongst those JSDoc warnings, I'll update and/or create a new issue to address this

--- a/blocks/autocompleters/index.js
+++ b/blocks/autocompleters/index.js
@@ -118,12 +118,12 @@ export function userAutocompleter() {
 		} );
 	};
 
-	const allowNode = ( textNode ) => {
+	const allowNode = () => {
 		return true;
 	};
 
 	const onSelect = ( user ) => {
-		return <span class="wp-user-mention">{ '@' + user.slug }</span>;
+		return <span className="wp-user-mention">{ '@' + user.slug }</span>;
 	};
 
 	return {

--- a/blocks/autocompleters/index.js
+++ b/blocks/autocompleters/index.js
@@ -123,7 +123,7 @@ export function userAutocompleter() {
 	};
 
 	const onSelect = ( user ) => {
-		return <span className="wp-user-mention">{ '@' + user.slug }</span>;
+		return ( '@' + user.slug );
 	};
 
 	return {

